README.md

```diff
@@ -279,8 +279,17 @@ You can specify media shortcuts through the `shortcuts` option, either as a list
       - icon: mdi:dog
         type: script
         id: script.script_name
+      - name: Crooners
+        type: service
+        id: spotcast.start
+        data: {
+          "entity_id" : "media_player.googlehome1234",
+          "uri" : "spotify:playlist:37i9dQZF1DX9XiAcF7t1s5"
+        }
```

Comment:
I would prefer if we specified the data like this:
```yaml
data:
  entity_id: media_player.googlehome1234
  uri: spotify:playlist:37i9dQZF1DX9XiAcF7t1s5
```

Reply:
Ofc! I was to focused on the `data_template` options by editing this